deepspeed/model_implementations/transformers/ds_transformer.py

```diff
@@ -13,9 +13,6 @@
 from deepspeed.accelerator import get_accelerator
 from deepspeed.ops.op_builder import InferenceBuilder
 import deepspeed
-if deepspeed.HAS_TRITON:
```

Comment:
@eonsparks - could you please run the pre-commit formatter (`pre-commit run --all-files`) to resolve te formatting errors?